### PR TITLE
Chloe feature account view

### PR DESCRIPTION
**Pull request by Chloe-Meinshausen**
Work In Progress not ready for Review yet

### CODE REVIEW

File: client/src/account/ListingList.jsx [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+// }
+
+// export default () =>
+//   <BootstrapTable keyField='id' data={ products } columns={ columns } />
+
+export default function ListingList (props) {
+  var items = props.items;
+  const products = props.transactions;
+
+  console.log('props', props);
```
Comment (nickgerrard) [inline client/src/account/ListingList.jsx]: did you want to remove this console.log?

File: client/src/account/ListingTabs.jsx [hunk 2]
```diff
@@ -0,0 +1,67 @@
+import Tab from 'react-bootstrap/Tab';
+import Tabs from 'react-bootstrap/Tabs';
+import React, {useState} from 'react';
+import TabContent from 'react-bootstrap/TabContent';
```
Comment (nickgerrard) [inline client/src/account/ListingTabs.jsx]: doesn't look like this is used?

File: client/src/account/ListingList.jsx [hunk 3]
```diff
@@ -0,0 +1,58 @@
+import ListGroup from 'react-bootstrap/ListGroup';
+import React from 'react';
+import Table from 'react-bootstrap/Table';
+
+// var transactions = require('../data/dataFunctions/transactions.json');
+
+const columns = [{
```
Comment (nickgerrard) [inline client/src/account/ListingList.jsx]: it doesn't look like columns is being passed to your table at all. Maybe I'm missing something here?

**Comment by joeyohie** [comment]
Wow! I really like all the sorting features for each column and the styling with every other row being grey or white.

File: client/src/account/RentalTabs.jsx [hunk 4]
```diff
@@ -4,35 +4,106 @@ import React, {useState} from 'react';
...
+  },
+  {
+    dataField: 'photos',
+    text: 'image'
+  }];
+
+  var mapper = (transactions, items) => {
+    var pastRentals = [];
+    var currentRentals = [];
+    items = items;
```
Comment (nickgerrard) [inline client/src/account/RentalTabs.jsx:59]: did you mean to declare this as a new variable?

**Review by nickgerrard — COMMENTED** [review]
it looks good! I left a few comments, mostly regarding code that looks like it isn't being used at alll. There's a chance I was misreading some of it though. I did notice that when you sort the columns, the first click doesn't quite work, but once you click a second time, it starts to sort really well! I'm happy to try and figure out why this is happening if you are experiencing it as well, but a first look, I'm at a loss.

**Comment by Chloe-Meinshausen** [comment]
THANK you nick! for those comments! they are very helpful.  Yeah I kind of gave up on the sort for now.   I can't figure out why it doesn't work and I'm afraid it will suck up too much time.   haha I may have fantasized  about clicking twice very quickly during the demonstrations. ;)

**Review by nickgerrard — APPROVED** [review]
looks good! I'm also thinking Async/Await may fix your sorting delay!

**Pull request merged**